src/UI/Cells/NzbDroneCell.js

```diff
@@ -18,6 +19,25 @@ module.exports = Backgrid.Cell.extend({
                 }
             });
         }
+
+        this.render = _.wrap(this.render, this._renderDecorator);
+    },
+
+    _renderDecorator : function(renderInner) {
+        var el = this.$el;
+
+        if (this.$el.hasClass('editable')) {
```

Comment:
one problem is that editability in backgrid is determined by the column, not the cell.